frontend/pages/beta/index.tsx

```diff
@@ -0,0 +1,164 @@
+import SearchIcon from '@mui/icons-material/Search'
+import Box from '@mui/material/Box'
+import Chip from '@mui/material/Chip'
+import Grid from '@mui/material/Grid'
+import IconButton from '@mui/material/IconButton'
+import InputBase from '@mui/material/InputBase'
+import MuiLink from '@mui/material/Link'
+import Paper from '@mui/material/Paper'
+import Stack from '@mui/material/Stack'
+import { useTheme } from '@mui/material/styles'
+import Tab from '@mui/material/Tab'
+import Tabs from '@mui/material/Tabs'
+import Typography from '@mui/material/Typography'
+import Link from 'next/link'
+import React, { useState } from 'react'
+import TagSelector from 'src/common/TagSelector'
+
+import { ListModelType, useListModels } from '../../data/model'
+import EmptyBlob from '../../src/common/EmptyBlob'
+import MultipleErrorWrapper from '../../src/errors/MultipleErrorWrapper'
+import Wrapper from '../../src/Wrapper'
+import { Model, Version } from '../../types/types'
+import useDebounce from '../../utils/hooks/useDebounce'
+
+export default function ExploreModels() {
+  const [group, setGroup] = useState<ListModelType>('all')
+  const [filter, setFilter] = useState('')
+  const debouncedFilter = useDebounce(filter, 250)
+
+  const { models, isModelsError, mutateModels } = useListModels(group, debouncedFilter)
+
+  const theme = useTheme()
+
+  const error = MultipleErrorWrapper(`Unable to load marketplace page`, {
```

Comment:
This is what causes multiple Wrapper components to be rendered on the page when we display errors. It's probably worth fixing this all in one go in a separate ticket, but just an FYI that any further uses of `MultipleErrorWrapper` like this is adding more work to that ticket.